Add webp to list of allowed extensions.
I have first tried to set this in the di.xml of this module, but for some reason Magento ignored that.
===================================================================
--- a/etc/di.xml
+++ b/etc/di.xml	(date 1737130764945)
@@ -38,12 +38,14 @@
                     <item name="jpeg" xsi:type="string">image/jpeg</item>
                     <item name="png" xsi:type="string">image/png</item>
                     <item name="gif" xsi:type="string">image/gif</item>
+                    <item name="webp" xsi:type="string">image/webp</item>
                 </item>
                 <item name="image_allowed" xsi:type="array">
                     <item name="jpg" xsi:type="string">image/jpg</item>
                     <item name="jpeg" xsi:type="string">image/jpeg</item>
                     <item name="png" xsi:type="string">image/png</item>
                     <item name="gif" xsi:type="string">image/gif</item>
+                    <item name="webp" xsi:type="string">image/webp</item>
                 </item>
                 <item name="media_allowed" xsi:type="array">
                     <item name="flv" xsi:type="string">video/x-flv</item>
